### ISSUE TITLE
IMDB Ranks

### ISSUE THREAD

**Issue by ShibajiCh** [jurialmunkey/skin.arctic.horizon.2#54]
The IMDB Top 250 Movies and TV Shows are not displaying their ranks in the info image above posters. Could you please check?
![IMG20220605223448](https://user-images.githubusercontent.com/65297828/172061903-0f54f9f5-0dd1-4342-a5c5-00dfeec9e2de.jpg)
![IMG20220605223511](https://user-images.githubusercontent.com/65297828/172061908-2c8bd4a8-3c20-4169-a0a4-725cf7c5cd32.jpg)

**Comment by ShibajiCh** [comment]
> Option to hide empty ratings will be coming.

Hi @jurialmunkey,

Glad to hear that. Thank you so much! I'll be waiting for that feature. 🙂👍🏻

Thanks and Regards,
ShibajiCh.

**Comment by jurialmunkey** [comment]
> Been playing around a little bit with the ratings ,I do see a plus side to this just notice the ratings now don't fade in they show up instantly which is actually way BETTER

Yeah there's benefits to both approaches. With the placeholder, we don't have to wait for the screen render to update the visibility condition so it can feel a touch faster since it is just the label updating the value. Plus the placeholder means a consistent position on screen.

Definitely benefits to hiding too though, so there's an option in 0.2.1 in the ratings config to disable the placeholder ratings and restore the old behaviour.

**Comment by ShibajiCh** [comment]
Thank you!
